analyzer/src/analysis/memory_location.cpp

```diff
@@ -65,7 +65,9 @@ LocalMemoryLocation::LocalMemoryLocation(ar::LocalVariable* var)
 }
 
 void LocalMemoryLocation::dump(std::ostream& o) const {
+  o << "LML{";
```

Comment:
being nitpicky here: could we use the full name here? same below
```suggestion
  o << "LocalMemoryLocation{";
```

Reply:
Will do.